src/widgets/Window.cpp

```diff
@@ -351,6 +351,11 @@ void Window::addShortcuts()
             getApp()->twitch.server->getOrAddChannel(si.channelName));
         splitContainer->appendSplit(split);
     });
+
+    createWindowShortcut(this, "CTRL+H", [this] {
+        getSettings()->hideSimilar.setValue(!getSettings()->hideSimilar);
+        getApp()->windows->forceLayoutChannelViews();
+    });
```

Comment:
Like I've commented before, I dislike a button to enable or disable this. This could be added when there is proper custom support for keyboard shortcuts. 
Right now the issue is if someone accidently presses this and wonders why the chat doesn't show all messages or is really slow since there is no indication at all that any message is being removed.

Reply:
You can't accidentally press this button, since this button in default state does nothing, you will have to enable a setting to hide similar messages which does in the name tell you about this ctrl+h shortcut

i find this shortcut extremely useful